website/thaliawebsite/templates/paginated_page.html

```diff
@@ -0,0 +1,33 @@
+{% extends 'simple_page.html' %}
+{% load django_bootstrap5 %}
+
+{% block pages %}
+<nav>
+    <ul class="pagination mt-4">
+        {% if page_obj.has_previous %}
+            <li class="page-item">
+                <a class="page-link"
+                href="{{ request.path }}?page={{ photos.previous_page_number }}">
+                <span aria-hidden="true">&laquo;</span>
+                <span class="sr-only">Previous</span>
+            </a>
+        </li>
+        {% endif %}
+        {% for page in page_range %}
+            <li class="page-item{% if page == page_obj.number %} active{% endif %}">
+                <a class="page-link" href="{{ request.path }}?page={{ page }}">{{ page }}
+                </a>
+            </li>
+        {% endfor %}
+        {% if page_obj.has_next %}
+            <li class="page-item">
+                <a class="page-link"
+                href="{{ request.path }}?page={{ photos.next_page_number }}">
```

Comment:
The `photos.next_page_number` looks not so generic. (same above)

I think it would be nice to have PagedView.get_context_data just provide e.g. `next_page`,  and `previous_page` (`_number` and/or `_url`).